build.gradle

```diff
@@ -6,17 +6,42 @@ apply plugin: 'com.gradle.build-scan'
 
 // Adds support for Node.js scripts
 apply plugin: 'com.moowork.node'
+apply plugin: 'com.wiredforcode.spawn'
+
+//NOTE: The startHsql task can be executed after uportal-war is built because it uses hsqldb-2.3.2.jar in classpath, otherwise it will fail
+task startHsql(type: SpawnProcessTask) {
+	description "Start hsql server"
+	def cmd = "java -cp ./uportal-war/target/uPortal/WEB-INF/lib/hsqldb-2.3.2.jar org.hsqldb.server.Server --database.0 file:./data/uPortal;hsqldb.tx=mvcc --dbname.0 uPortal --address localhost --port 8887"
+	command cmd
+	ready 'Started Application'
+}
```

Comment:
Ready value should be 'Startup sequence completed' rather than 'Started Application'